src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    private static final FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
+    private static final int NUMBER_OF_REPEAT = 4;
+    private static final Figure [] figures = new Figure[NUMBER_OF_REPEAT];
+    private static final int MIDDLE = figures.length / 2;
+
+    public static void main(String[] args) {
+        for (int i = 0; i < MIDDLE;i++) {
+            figures[i] = FIGURE_SUPPLIER.getRandomFigure();
+            figures[i].draw();
+        }
+        for (int i = MIDDLE;i < NUMBER_OF_REPEAT;i++) {
+            figures[i] = FIGURE_SUPPLIER.getDefaultFigure();
+            figures[i].draw();
+        }
+    }

```

Comment:
2 minor points here: 
1) Please pay attention to the task description: "**After generating the array**, we need to display the entire list of objects...:"
2) Please put a space before "i++" in both cases and a space before "i < NUMBER_OF_REPEAT"